packages/service-worker/src/low_level.ts

```diff
@@ -46,6 +46,19 @@ export interface UpdateActivatedEvent {
   current: {hash: string, appData?: Object};
 }
 
+/**
+ * An event emitted when the service worker has checked the version of the app on the server
+ * and it is the same as the installed version.
+ *
+ * @see {@link guide/service-worker-communications Service worker communication guide}
+ *
+ * @publicApi
+ */
+export interface NoNewVersionDetectedEvent {
+  type: 'NO_NEW_VERSION_DETECTED';
+  version: {hash: string; appData?: object;};
```

Comment:
Should it look like this instead (similar to other events)?
```suggestion
  version: {hash: string; appData?: Object};
```

Reply:
TS team (I believe) discourages that form that btw. I'm surprised there's no linting in this repo to prefer `string`, `number`, `object`, etc

Reply:
That's right - this is part of the Google TS style guide as well https://google.github.io/styleguide/tsguide.html#wrapper-types